src/sql/src/plan/lowering/variadic_left.rs

```diff
@@ -255,12 +262,17 @@ pub(crate) fn attempt_left_join_magic(
             // so that after the join is done we can overwrite all values for `right` with null values.
             // This is a quirk of how outer joins work: the matched columns are left as null.
 
+            let aug_value = MirRelationExpr::union(
+                bindings[index + 1].1.clone().filter(
```

Comment:
I feel like this can be simplified because `bindings[index + 1].1.clone()` will always be equal to `get_right` here, so one can probably already construct `get_right` to have the filter on top here

```rust
let get_right = MirRelationExpr::local_get(id, right_type.clone());
```

or sometime later, probably immediately after

```rust
let equations = if let Some(list) = decompose_equations(&on) {
    list
}
```

as it seems that `bindings` doesn't ever read the second position of the value tuple:

```rust
while let Some((id, _get, value)) = bindings.pop() {

}
```